Documentation/Scripts/codeBlockReader.py

```diff
@@ -264,33 +260,40 @@ def example_content(filepath, fh, tag, blockType, placeIntoFilePath):
   fh.write("</div>\n")
   fh.write("\n")
 
-
-def fetch_comments(dirpath, forceDokuBlockContent):
+def fetch_comments(dirpath, forceDocuBlockContent):
   """ Fetches comments from files and writes to a file in required format.
   """
   global fullSuccess
-  global validExtensions
   comments_filename = "allComments.txt"
   fh = io.open(comments_filename, "a", encoding="utf-8", newline="")
   shouldIgnoreLine = False
 
   for root, directories, files in os.walk(dirpath):
     for filename in files:
```

Comment:
Cannot rely on alphabetical order. Seems to work on Windows but not on Linux (`1_structs.md` somewhere in the middle instead of first).
```suggestion
    for filename in sorted(files):
```